README.md

```diff
@@ -58,10 +58,16 @@ Raspberry Pi 2/3/4 with GNU/Linux.
   is necessary to have Arm/Linux installed in your build environment.
 * Install [GNU Toolchain for the A-profile Architecture](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-a/downloads)
     - Select `arm-linux-none-gnueabihf` (AArch32 target with hard float)
+```shell 
+$ sudo apt-get install gcc-arm-linux-gnueabihf 
+
+# set QEMU_LD_PREFIX path
+$ export QEMU_LD_PREFIX=/usr/arm-linux-gnueabihf
```

Comment:
Don't rely on manual environment setting. Instead, improve the build system to validate Arm toolchain installation. If the GNU toolchain was downloaded and extracted from Arm website, the environment variable must point to different directories.

Reply:
Not considering manual environment setting, and cannot set shell environment variable from `make` (Linux shell limitation, I understand), can only think of `make qemu ./amacc tests/hello.c` kind of implementation. Not sure if that is a right approach?

Reply:
Someone (at least for me) might install a bunch of GNU toolchains and/or QEMU packages, and directory `/usr/arm-linux-gnueabihf` would not be always correct per installation. At present, [mk/arm.mk](https://github.com/jserv/amacc/blob/master/mk/arm.mk) manages to check the corresponding files once `$PATH` is set properly. You can extend the existing checks.

Reply:
Right, and `mk/arm.mk` also checks `$QEMU_LD_PREFIX` as well. It works and gets the LD path on my system (but not passing to `qemu`), can look into that in more details. With this PR, trying to fix is different. It is to fix the `$QEMU_LD_PREFIX` path issue on shell when executing `qemu-arm`.

Reply:
Again, let the build system detect the runtime libraries, and you can send pull requests to consolidate it if you found its shortcomings.